src/components/TimeseriesDataExport/stories/basic.story.mdx

```diff
@@ -0,0 +1,150 @@
+import { Meta, Story, Preview, Props } from '@storybook/addon-docs/blocks';
+import { TimeseriesDataExport, TimeseriesDataExportFC } from '../TimeseriesDataExport';
+import { 
+  decorators,
+  strings,
+  TimeseriesChartExportWrapper,
+} from './helper';
+
+<Meta title="Time Series|TimeseriesDataExport" decorators={decorators} />
+
+# Timeseries Data Export
+
+Component allows to export CSV data for provided timeseries in selected timerange with defined granularity.
+
+Component based on antd modal component with form, where user can define all configurations for CSV export.
+
+Component has embedded logic for fetching timeseries data and generating CSV file. To make it work properly, component
+has to be wrapped into `ClientSDKProvider` component with defined client SDK instance. Fetching logic
+also can be redefined via `retrieveTimeseries` and `fetchCSV` props.
+
+Imports:
+
+```jsx
+import 'antd/dist/antd.css';
+
+import React from 'react';
+import { TimeseriesDataExport } from '@cognite/gearbox';
+```
+we use small functional component for demostration.
+
+```jsx
+const TimeseriesChartExportWrapper: React.FC<
+  Omit<TimeseriesDataExportProps, 'visible' | 'form'>
+> = props => {
+  const [open, setOpen] = useState(false);
+  const onClose = () => setOpen(false);
+  const onOpen = () => setOpen(true);
+
+  return (
+    <>
+      <button onClick={onOpen}>Export Chart Data</button>
+      <TimeseriesDataExport visible={open} hideModal={onClose} {...props} />
+    </>
+  );
+};
+```
+
+<Preview>
+  <Story name="Basic Usage" >
+    <TimeseriesChartExportWrapper
+      timeseriesIds={[41852231325889, 7433885982156917]}
+      granularity={'2m'}
+      defaultTimeRange={[1567321800000, 1567408200000]}
+    />
+  </Story>
+</Preview>
+
+## Available props
+
+<Props of={TimeseriesDataExportFC} />
```

Comment:
<img width="1180" alt="Screenshot 2019-12-23 at 10 23 00" src="https://user-images.githubusercontent.com/24567091/71345539-591e2700-256e-11ea-9a35-3b7fb4e6459b.png">
this 2 prop comes from HOC which is redundant knowledge for the user. Could you please just add a comment below Props section that user shouldn't specify these 2 props. We will later figure out how to hide it at all